Code/GraphMol/FMCS/FMCS.h

```diff
@@ -90,26 +102,5 @@ namespace RDKit {
     void      parseMCSParametersJSON (const char* json, MCSParameters* params);
     MCSResult findMCS (const std::vector<ROMOL_SPTR>& mols, const MCSParameters* params=0);
 
-    typedef enum {
-        AtomCompareAny,
-        AtomCompareElements,
-        AtomCompareIsotopes
-    } AtomComparator;
-    typedef enum {
-        BondCompareAny,
-        BondCompareOrder,
-        BondCompareOrderExact
-    } BondComparator;
-    MCSResult findMCS (const std::vector<ROMOL_SPTR>& mols,
```

Comment:
I'm ok with this function being removed from the public interface, but doing so causes the Java tests to break (they use this function). If you're going to remove it, please ensure that the java wrappers continue to work and that those tests pass.